modules/tensor_mechanics/test/tests/visco/single_element_model_elastic.i

```diff
@@ -0,0 +1,209 @@
+  [../]
+[]
+
+[BCs]
+  [./symmy]
+    type = DirichletBC
+    variable = disp_y
+    boundary = bottom
+    value = 0
+  [../]
+  [./symmx]
+    type = DirichletBC
+    variable = disp_x
+    boundary = left
+    value = 0
+  [../]
+  [./symmz]
+    type = DirichletBC
+    variable = disp_z
+    boundary = back
+    value = 0
+  [../]
+  [./axial_load]
+    type = FunctionDirichletBC
+    variable = disp_x
+    boundary = right
+    function = strain_history
+  [../]
+[]
+
+[Materials]
+  # [./burgers]
+  #   type = GeneralizedKelvinVoigtModel
+  #   creep_modulus = '   1.52e06
+  #                       5.32e12
+  #                       6.15e04
+  #                       6.86e04
+  #                       4.48e04
+  #                       1.05e12' # data from TAMU
+  #   creep_viscosity = ' 1
+  #                       10
+  #                       100
+  #                       1000
+  #                       10000
+  #                       100000'  # data from TAMU
+  #   poisson_ratio = 0.2
+  #   young_modulus = 33570.84903
+  # [../]
+  # [./stress]
+  #   type = ComputeMultipleInelasticStress
+  #   inelastic_models = 'creep'
+  # [../]
+  [./stress]
+    type = ComputeLinearElasticStress
+    elasticity_tensor = 33570.84903
+    # inelastic_models = 'creep'
+  [../]
+  # [./creep]
+  #   type = LinearViscoelasticStressUpdate
+  # [../]
+  [./strain]
+    type = ComputeIncrementalSmallStrain
+    displacements = 'disp_x disp_y disp_z'
+  [../]
+[]
+
+# [UserObjects]
+#   [./update]
+#     type = LinearViscoelasticityManager
+#     viscoelastic_model = burgers
+#   [../]
+# []
+
+[Postprocessors]
+  [./stress_xx]
+    type = ElementAverageValue
+    variable = stress_xx
+    block = 'ANY_BLOCK_ID 0'
+  [../]
+  [./strain_xx]
+    type = ElementAverageValue
+    variable = strain_xx
+    block = 'ANY_BLOCK_ID 0'
+  [../]
+  # [./creep_strain_xx]
+  #   type = ElementAverageValue
+  #   variable = creep_strain_xx
+  #   block = 'ANY_BLOCK_ID 0'
+  # [../]
+[]
+
+[Preconditioning]
+  [./smp]
+    type = SMP
+    full = true
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+
+  l_max_its  = 50
+  l_tol      = 1e-10
+  nl_max_its = 20
+  nl_rel_tol = 1e-10
+  nl_abs_tol = 1e-10
+
+  dtmin = 0.01
+  end_time = 17000
+  [./TimeStepper]
+    type = LogConstantDT
+    first_dt = 0.1
+    log_dt = 0.1
+  [../]
+[]
+
+[Outputs]
+  # file_base = single_element_creep_out
+  # exodus = true
+
+ perf_graph     = true
+ csv = true
+ [./Console]
+   type = Console
+ [../]
+ [./Exo]
+   type = Exodus
+   elemental_as_nodal = true
+ [../]
```

Comment:
can you please `hit format` all those inputs

Reply:
This file is no longer in this commit.